documentation/source/install/cloud_solutions.rst

```diff
@@ -12,7 +12,7 @@ Setting up OpenEBS with Kubernetes on Amazon EC2 Instances
 
```

Comment:
Amazon EC2 Instances or Amazon Web Services? Can you get it confirmed from Kiran.

Reply:
Changed to Amazon Web Services